README.md

```diff
@@ -12,13 +12,13 @@ This repo features:
 
 ## Contributing to this repo
 
-* Request access from a member of [the Data Science team](https://the-strategy-unit.github.io/data_science/about.html), they need to add you as a contributor
+* Request access from a member of [the Data Science team](https://the-strategy-unit.github.io/data_science/about.html), they need to add you as a contributor. Alternatively, you can [fork the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) to your own GitHub account.
```

Comment:
Could this section read more like a numbered step by step? Might read like you can fork (this bullet) and then also clone (bullet 3).

Reply:
If you fork it you do still need to clone the forked repo to work on it on your local machine though!

Reply:
For sure, but I wondered if the current wording might sound like 'fork The-Strategy-Unit/data_science and then clone The-Strategy-Unit/data_science' to folks less acquainted with Git/GitHub? Happy to let this go, since the intention of this README isn't to teach people how to use Git/GitHub, of course!